packages/cli-test/src/cli/commands/datastore.ts

```diff
@@ -0,0 +1,82 @@
+import type { ProjectCommandArguments } from '../../types/commands/common_arguments';
+import { SlackCLIProcess } from '../cli-process';
+
+export interface DatastoreCommandArguments {
+  /** @description datastore get <query> */
+  getQuery: string;
```

Comment:
Here I'm using general string for datastore commands to make it easier than pre-define attributes such as `datastore`, `item`, and `expression`, etc

Reply:
I feel like this will make it easy to make mistakes. I think we should model these just like we model the actual methods:
- [`apps.datastore.get` method](https://api.slack.com/methods/apps.datastore.get) needs `datastore` string and `id` string
- [`apps.datastore.query` method](https://api.slack.com/methods/apps.datastore.query) needs `datastore` string, and its optional arguments, etc.

Reply:
It makes sense, I'd say for `get` method, `id` is not a must as it's defined in the manifest as primary key. I'd like to change by adding the `datastore` as the new argument here.